README.md

```diff
@@ -0,0 +1,72 @@
+# My INF 7420 Project
+I am going to build a site to support a youth literacy initiative project that a colleague and I are currently working on. 
+
+## Wireframes
+Here are the wireframes for the projects home, about, resources, contact and social media pages. 
+
+### Home Page
+
+![wireframe of Home Page](wireframes/home.jpg)
```

Comment:
I should have clarified that the references to files are case-sensitive--it should match the exact case as your directory name and your file name. Your file name is `Home.jpg` and not `home.jpg` so this is also incorrect. You can choose to change the name of the image file so that it is lowercase to match this reference or you can change this reference to match the exact case of the image file name.

Reply:
This issue applies to all images in the README.md file

Reply:
The file names have been updated to match the lowercase. Thank you.

Reply:
@nicolelynw I don't see this change. Was there a commit that was not pushed?